s2i v1.5.0 Fails to Build for Windows

When trying to build v1.5.0 binaries for Windows, they fail with the following error:

```
# github.com/Microsoft/hcsshim/internal/oc
vendor/github.com/Microsoft/hcsshim/internal/oc/errors.go:21:39: undefined: errdefs.ToGRPC
```

This does not appear to be an issue for `linux` and `darwin` OS platforms.

--- a/go.mod
+++ b/go.mod
@@ -21,9 +21,10 @@ require (
 	github.com/Azure/go-ansiterm v0.0.0-20230124172434-306776ec8161 // indirect
 	github.com/BurntSushi/toml v1.4.0 // indirect
 	github.com/Microsoft/go-winio v0.6.2 // indirect
-	github.com/Microsoft/hcsshim v0.12.7 // indirect
+	github.com/Microsoft/hcsshim v0.12.9 // indirect
 	github.com/containerd/cgroups/v3 v3.0.3 // indirect
-	github.com/containerd/errdefs v0.2.0 // indirect
+	github.com/containerd/errdefs v0.3.0 // indirect
+	github.com/containerd/errdefs/pkg v0.3.0 // indirect
 	github.com/containerd/stargz-snapshotter/estargz v0.15.1 // indirect
 	github.com/containerd/typeurl/v2 v2.2.0 // indirect
 	github.com/containers/libtrust v0.0.0-20230121012942-c1716e8a8d01 // indirect

